.gitignore

```diff
@@ -3,3 +3,5 @@
 /test-suite-externals
 *.pyc
 /build*
+.vscode/**
```

Comment:
the .gitignore file probably shouldn't be modified for everyone in this fashion.

I'm not sure why you'd have a .vscode file inside your repo directory. If using VSCode to "open" a directory, make sure it's the parent.